### PR TITLE
update History Log to React

### PR DESCRIPTION
**Pull request by kongtiaowang**
This pull request `[DESCRIBE WHAT THIS PULL REQUEST IS TRYING TO ACCOMPLISH HERE]`. It `[DESCRIBE HOW IT ACCOMPLISHES THE GOAL HERE]`.

See also: `[PROVIDE REFERENCES TO OTHER TICKETS, PULL REQUESTS, OR MAILING LIST THREADS HERE. OPTIONAL.]`

### CODE REVIEW

File: modules/final_radiological_review/ajax/log.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ ini_set('default_charset', 'utf-8');
+ $DB =& Database::singleton();
+
+   $headers = ['Time','User','Field','Old Value','New Value'];
+   
+            $history = $DB->pselect(
+                'SELECT  changeDate, userID, col, old, new 
+                 FROM final_radiological_review_history
+                WHERE CommentID= :primKey ORDER BY changeDate DESC',
+                array('primKey' => '300022MTL02222211468524343')
```
Comment (driusan) [inline modules/final_radiological_review/ajax/log.php]: I'm fairly sure this isn't supposed to be a hardcoded CommentID...

(This file also looks like it hasn't been run through PHPCS? The indentation is all off.)

File: modules/final_radiological_review/templates/form_final_radiological_review.tpl [hunk 2]
```diff
@@ -112,18 +112,14 @@
...
-        </thead>
-        <tbody>
-            {$form.history.html}
-        </tbody>
-    </table>
+<div class="dynamictable" id="datatable"></div>
+<script>
+loris.hiddenHeaders = {(empty($hiddenHeaders))? [] : $hiddenHeaders };
+var table = RDynamicDataTable({
+     "DataURL" : "{$baseurl}/final_radiological_review/ajax/log.php?identifier=" + $identifier,
```
Comment (jstirling91) [inline modules/final_radiological_review/templates/form_final_radiological_review.tpl]: `$identifier` needs to be in `{ }` in order for the smarty to pass the value

File: modules/final_radiological_review/php/NDB_Menu_Filter_final_radiological_review.class.inc [hunk 3]
```diff
@@ -416,6 +416,7 @@ class NDB_Menu_Filter_Final_Radiological_Review extends NDB_Menu_Filter
         return array_merge(
             $deps,
             array(
+             $baseURL . "/final_radiological_review/js/columnFormatter_HistoryLog.js",
```
Comment (jstirling91) [inline modules/final_radiological_review/php/NDB_Menu_Filter_final_radiological_review.class.inc]: This needs to go in `NDB_Form_final_radiological_review.class.inc`. You'll also need to add 
```
$baseURL . '/js/components/StaticDataTable.js',
$baseURL . '/js/components/DynamicDataTable.js',
```

File: modules/final_radiological_review/jsx/columnFormatter_HistoryLog.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+/* exported formatColumn */
```
Comment (jstirling91) [inline modules/final_radiological_review/jsx/columnFormatter_HistoryLog.js]: You're missing the compiled version of this

**Pull request closed without merging**